cli/base.ts

```diff
@@ -47,7 +47,7 @@ export abstract class BaseCommand extends Command {
     if (date) {
```

Comment:
Per discussion over hangouts, it would probably be cleanest to perform these calculations all in UTC. We can still accept input in different timezones but should translate them to UTC before hitting this function.